src/logger/loggerApi.c

```diff
@@ -1,58 +1,61 @@
-/**
- * AutoSport Labs - Race Capture Pro Firmware
+/*
+ * Race Capture Pro Firmware
  *
- * Copyright (C) 2014 AutoSport Labs
+ * Copyright (C) 2015 Autosport Labs
  *
- * This file is part of the Race Capture Pro firmware suite
+ * This file is part of the Race Capture Pro fimrware suite
```

Comment:
Ooohhh... yeah that is from a template... which means its everywhere :(.  Damnit.  Will fix separately.